doc/manual/manual.tex

```diff
@@ -7073,7 +7073,68 @@ \subsubsection{Melt migration in a 2D mid-ocean ridge model}
 \end{itemize}
 
 
+\subsubsection{Kinematically-driven 2d oceanic subduction}
+\label{sec:cookbooks-kinematic-2d-oceanic-subduction}
 
+\textit{This section was contributed by Anne Glerum.}
+
+This subduction example is based on the benchmark effort of Quinquis et al., of which initial results were published in \cite{Quinquis2014}. In 4 increasingly complex setups we will go from isoviscous materials without any temperature effects and to a fully thermo-mechanical, nonlinear, strain-weakened visco-plastic, externally-driven model of oceanic subduction. The setup of the most complex case is outlined in Fig.~\ref{fig:QQ_setup}. The models are run for 15 My and slab tip depth, trench location, RMS velocity and temperature, and viscous dissipation are compared between different codes. In addition, we discuss the effects of the element size of the subduction interface and crustal layers, viscosity averaging and the solver tolerance.
```

Comment:
```suggestion
This subduction example is based on the benchmark effort of Quinquis et al., of which initial results were published in \cite{Quinquis2014}. In four increasingly complex setups we will go from isoviscous materials without any temperature effects to a fully thermo-mechanical, nonlinear, strain-weakened visco-plastic, externally-driven model of oceanic subduction. The setup of the most complex case is outlined in Fig.~\ref{fig:QQ_setup}. The models are run for 15 My and slab tip depth, trench location, RMS velocity and temperature, and viscous dissipation are compared between different codes. In addition, we discuss the effects of the element size of the subduction interface and crustal layers, viscosity averaging and the solver tolerance.
```

Reply:
Are you really going to compare results between different codes? Or, if not, maybe change the wording to say that the comparison is in Quinquis2014